src/utils/userUid.js

```diff
@@ -0,0 +1,12 @@
+import jwt_decode from 'jwt-decode'
+
+const session = JSON.parse(localStorage.getItem('session'))
```

Comment:
Instead of fetching the session directly from local storage, use this function https://github.com/code-mtaani/ojwang-ui/blob/8a5f92ee71600116923adab81086c313019568ea/src/utils/auth.js#L41 that way, I will be easier to make changes if we decide to change the location or the name of the session.

Also, can we move this method to src/utils/auth.js ? other similar functions are there.

Reply:
okay. I'll do that.

Reply:
Done

Reply:
I suggest we have all auth-related helpers in one file for now and refactor later if necessary.